back-end/src/main/java/index/PsqlNativeBoxIndexer.java

```diff
@@ -0,0 +1,434 @@
+package index;
+
+import jdk.nashorn.api.scripting.NashornScriptEngine;
+import main.Config;
+import main.DbConnector;
+import main.Main;
+import project.Canvas;
+import project.Layer;
+import project.Transform;
+import box.Box;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.Date;
+import org.postgresql.PGConnection;
+import org.postgresql.copy.CopyManager;
+import java.io.StringReader;
+import java.util.function.*;
+
+/**
+ * Created by wenbo on 12/30/18.
```

Comment:
let's change it or remove it...

Reply:
DONE/sorry - removed. Given "git blame" there's little reason to mention the original author... ;-)